MapStory/mapstory
# Re-persisting API selection/toggle on explore page

Closes #1214 

To test:
- do whatever you do to update your JS assets
- go to explore page
- select "storytellers"
- refresh page
- url & ui toggle should stay on storyteller api
 - url should not contain `&content` or `&is_published`


Prior behavior(issue)
<img width="870" alt="screen shot 2018-03-05 at 1 08 55 pm" src="https://user-images.githubusercontent.com/15912063/36994371-8897f136-2076-11e8-95fa-b8b18ab5e53e.png">

Expected behavior(fix)
<img width="1064" alt="screen shot 2018-03-05 at 1 07 31 pm" src="https://user-images.githubusercontent.com/15912063/36994379-8bfe2caa-2076-11e8-805d-1a3a9d1ae260.png">
